api/src/client/dynamics/housing/DynamicsHousingPropertyInterestClient.ts

```diff
@@ -1,21 +1,30 @@
-import { HousingPropertyInterest, HousingPropertyInterestClient } from "@client/dynamics/housing/types";
+import {
+  HousingPropertyInterest,
+  HousingPropertyInterestClient,
+  HousingPropertyInterestResponse,
+} from "@client/dynamics/housing/types";
 import { LogWriterType } from "@libs/logWriter";
 import axios, { AxiosError } from "axios";
 
 export const DynamicsHousingPropertyInterestClient = (
   logWriter: LogWriterType,
   orgUrl: string
 ): HousingPropertyInterestClient => {
-  const getPropertyInterests = async (
+  const getPropertyInterest = async (
```

Comment:
Makes more sense with our updated understanding of Housing to only search for a single property interest per address